backend/app.test.js

```
@@ -37,3 +41,58 @@ describe("App.js - Route setup & middlewares", () => {
+    process.env.FRONTEND_URL = "http://frontend.example.com";
+    process.env.BACKEND_URL = "http://backend.example.com";
+
+    const { getSwaggerSpec } = await import("./config/swagger.js");
+    const swaggerSpec = getSwaggerSpec();
+
+    expect(swaggerSpec.info.contact.url).toBe("http://frontend.example.com");
+    expect(swaggerSpec.servers[0].url).toBe("http://backend.example.com");
+  });
+
+  it("should fall back to default values when FRONTEND_URL and BACKEND_URL are not set", async () => {
+    delete process.env.FRONTEND_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)